xml/System.Diagnostics/ActivityEvent.xml

```diff
@@ -34,8 +34,8 @@
         <Parameter Name="name" Type="System.String" />
       </Parameters>
       <Docs>
-        <param name="name">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="name">The event name.</param>
+        <summary>Initializes a new activity event instance using the specified name.</summary>
```

Comment:
>Initializes a new activity event instance using the specified name. [](start = 17, length = 67)

Initializes a new activity event instance using the specified name and the current time as the event timestamp.

Reply:
```suggestion
        <summary>Initializes a new activity event instance using the specified name and the current time as the event timestamp.</summary>
```